configs/community.dbuild

```diff
@@ -1317,6 +1317,26 @@ build += {
     extra.exclude: ["plugin", "themePlugin", "genericTheme"]
   }
 
+  // there are a *ton* of subprojects.  it might be interesting to try to add
+  // as many as possible. for now, we've somewhat arbitrarily selected a few
+  ${vars.base} {
+    name: "lagom"
+    uri:  ${vars.uris.lagom-uri}
+    extra.projects: ["server", "testkit-scaladsl", "immutables"]
```

Comment:
Immutables is probably not very useful (it contains only a single Java annotation definition)

Reply:
okay, I took that out